Automattic/jetpack
# SSO: Add custom message to new user email

Fixes https://github.com/Automattic/wp-calypso/issues/86760

## Proposed changes:

* Adds a "Custom Message" field to the wp-admin new user form

**New user form:**

![image](https://github.com/Automattic/jetpack/assets/10482592/2b3b267b-c588-4162-bd8a-2c045240eeab)

**Email:**
<img width="788" alt="CleanShot 2024-02-01 at 10 40 34@2x" src="https://github.com/Automattic/jetpack/assets/10482592/ae86a7ad-9d07-4745-8894-92c9b877e78f">



### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






1. On an Atomic, JN, or whatever non-simple site, install Jetpack.
2. Make sure SSO is on, it should be on by default.
3. Sync this PR to your site.
4. Navigate to `/wp-admin/user-new.php`
5. Add a username, email, and custom message.
6. Verify the custom message is sent in email.
